### DISCUSSION TITLE
toYaml breaks the application-secrets.yml file

### DISCUSSION THREAD

**Discussion by mlongarai** [tchiotludo/akhq#1004] [Q&A] [answered]
Hey guys,

I just found out that the _toYaml_ convert positioned in the secrets template, is breaking the use of the "|" pipe when the pod is created.

`application-secrets.yml: {{ toYaml .Values.secrets | b64enc | quote }}`

From my tests it should be - at least worked for me - (without toYaml):
``application-secrets.yml: {{ .Values.secrets | b64enc | quote }}``

Could you guys fix this or it require more analysis? 

Thank you!

**Comment by tchiotludo** [comment, marked as answer]
It's was an intended behaviour, you need to enter values like that your values : 
```yaml
secrets: 
  your-secret: your-value
```` 
and not like that : 
```yaml
secrets: |
  your-secret: your-value
````

**Comment by mlongarai** [comment]
Solved, thank you!
